### ISSUE TITLE
Booting loop after installing sedutil

### ISSUE THREAD

**Issue by james645** [Drive-Trust-Alliance/sedutil#381]
I installed sedutil. After unlocking it shows "Booting OS" for 2 sec, then booting again, asks for password again.  --> loop

It does not find the installed linux OS. What can I do? 

BTW: after disablelockingrange and setMBREnable the linux is booting.

**Comment by r0m30** [comment]
If it's rebooting the PBA that means MBRdone was not properly set, did the PBA say that it unlocked the drive?￼ Another thing you will want to verify is that your bios isn't power cycling the drive on reboot

**Comment by james645** [comment]
yes PBA says its unlocked. I can't say for sure if bios is power cycling the drive, there's no option in bios (ami) and power led etc stay enlighted. disk is a samsung 980 pro. I read that sometimes about the path, I used /dev/nmve0, but I have two partitions, so maybe I have to use nvme0p1 instead?

**Comment by r0m30** [comment]
If the PBA says it's unlocked then it received return code 0 from both the setlockingrange and the setMBRDone. The only way that I know of for the disk to re-lock durring a reboot is with a power cycle of the drive. If your bios doesn't have anything to stop that there is currently no way to use the PBA on your machine.￼￼

**Comment by james645** [comment]
hm would be sad. I'll try some things.. Could there be a difference between using MBR or GPT partition table?

**Comment by james645** [comment]
there is something interesting: after fresh installation of manjaro (without sedutil/PBA) in bios there are two options in "UEFI NVME Drive BBS Priorities": 1) Manjaro 2) UEFI OS (Samsung 980)
After installing PBA there is only one option. Maybe it cannot boot manjaro after unlocking because bios cannot route to manjaro?

**Comment by alci63** [comment]
I experiment the same problem, with two Samsung disks (908por and 970evo). The installed OS is Ubuntu. Didn't find any setting in bios (AMI). I disabled drive locking for now...
